Postman/Survey/SurveySample.postman_collection.json

```diff
@@ -0,0 +1,361 @@
+						"{{SurveyId}}",
+						"Sample"
+					]
+				},
+				"description": "(Online) (Cati) Retrieves the sample data for the specified survey."
+			},
+			"response": []
+		},
+		{
+			"name": "Upload Sample",
+			"event": [
+				{
+					"listen": "test",
+					"script": {
+						"exec": [
+							"pm.test(\"Status code is 200\", function () {",
+							"    pm.response.to.have.status(200);",
+							"});",
+							"",
+							"const validateProperties = (item) => {",
+							"    return item.ProcessingStatus !== undefined",
+							"    && item.TotalRecordCount !== undefined",
+							"    && item.InsertedCount !== undefined",
+							"    && item.UpdatedCount !== undefined",
+							"    && item.DuplicateKeyCount !== undefined",
+							"    && item.EmptyKeyCount !== undefined",
+							"    && item.InvalidKeyCount !== undefined",
+							"    && item.InvalidDataCount !== undefined",
+							"    && item.SkippedCount !== undefined",
+							"    && item.HeaderInvalid !== undefined",
+							"    && item.HeaderInvalidColumnsCount !== undefined",
+							"    && item.HeaderDataMismatch !== undefined",
+							"    && item.ErrorMessages !== undefined;",
+							"}",
+							"",
+							"pm.test(\"Response contains All Properties\", function () {",
+							"    const response = pm.response.json();",
+							"    pm.expect(validateProperties(response)).to.be.true;",
+							"});"
+						],
+						"type": "text/javascript"
+					}
+				}
+			],
+			"request": {
+				"method": "POST",
+				"header": [
+					{
+						"key": "Content-Type",
+						"value": "application/json"
+					},
+					{
+						"key": "Authorization",
+						"value": "Basic {{AuthenticationToken}}"
+					}
+				],
+				"body": {
+					"mode": "file",
+					"file": {
+						"src": "/C:/repo_1/Nfield-SDK/Postman/Survey/SampleData.csv"
+					}
+				},
+				"url": {
+					"raw": "{{origin}}/v1/Surveys/{{SurveyId}}/Sample",
+					"host": [
+						"{{origin}}"
+					],
+					"path": [
+						"v1",
+						"Surveys",
+						"{{SurveyId}}",
+						"Sample"
+					]
+				},
+				"description": "(Online) (Cati) Uploads the sample data for the specified survey. Pass the sample data as a CSV formatted string in the body of the request."
+			},
+			"response": []
+		},
+		{
+			"name": "Delete Sample",
+			"event": [
+				{
+					"listen": "test",
+					"script": {
+						"exec": [
+							"pm.test(\"Status code is 204\", function () {",
+							"    pm.response.to.have.status(204);",
+							"});",
+							"",
+							""
+						],
+						"type": "text/javascript"
+					}
+				}
+			],
+			"request": {
+				"method": "DELETE",
+				"header": [
+					{
+						"key": "Content-Type",
+						"value": "application/json"
+					},
+					{
+						"key": "Authorization",
+						"value": "Basic {{AuthenticationToken}}"
+					}
+				],
+				"body": {
+					"mode": "raw",
+					"raw": "[\r\n  {\r\n    \"Name\": \"TelephoneNumber\",\r\n    \"Op\": \"Eq\",\r\n    \"Value\": \"empty\"\r\n  }\r\n]"
```

Comment:
Maybe TelephoneNumber could be a variable

Reply:
better not, because is a property name not variable value